src/SFA.DAS.LevyTransferMatching.Api/Controllers/PledgeController.cs

```diff
@@ -144,6 +146,22 @@ public async Task<IActionResult> ClosePledge(int pledgeId, [FromBody] SetClosePl
             }
         }
 
+        [HttpPost]
+        [Route("accounts/{accountId}/pledges/{pledgeId}/reject-applications")]
+        public async Task<IActionResult> RejectApplications(int accountId, int pledgeId, [FromBody] RejectApplicationsRequest request)
```

Comment:
Minor point, but methods with the same name and access level should be next to one another for clarity.

Reply:
Methos for RejectApplications with the same names are moved together now.